buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeImplPlugin.kt

```diff
@@ -359,7 +359,7 @@ private fun configureComposeCompilerPlugin(
         // Add Compose compiler plugin to kotlinPlugin configuration, making sure it works
```

Comment:
> Release Notes
> N/A

As we fix a bug, it should contain "Fixes - Multiplatform" "Fix IrLinkageError: Function can not be called: No function found for symbol"